pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -580,6 +592,9 @@ func (gc *GarbageCollector) worker() {
 	err := gc.processItem(key.(*node))
 	if err != nil {
 		utilruntime.HandleError(fmt.Errorf("Error syncing item %#v: %v", key, err))
+		// retry if garbage collection of an object failed.
+		gc.dirtyQueue.AddWithTimestamp(key, start)
```

Comment:
This is noise. I can separate it to another PR if you think the change is not obvious.